.eslintrc

```diff
@@ -0,0 +1,23 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "rules": {
+    "comma-dangle": ["error", "always-multiline"]
+  },
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": false
+    },
+    "project": "./tsconfig.json"
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ]
+}
```

Comment:
this project uses XO.

Reply:
done

Reply:
that's not what I mean, see https://github.com/egoist/vue-compile/blob/d5c00129d0d6043a1194c7ebd2353dbdb21c2d77/package.json#L51

Reply:
sorry, still can't work. I set: 
```json
"xo": {
    "extends": "xo-typescript",
    "extensions": [
      "ts"
    ],
    "rules": {
      "unicorn/filename-case": "off"
    }
  },
```
but report: `Cannot find module 'eslint-plugin-@typescript-eslint'`